gradle/libs.versions.toml

```diff
@@ -0,0 +1,6 @@
+[versions]
+slf4j = "2.0.7"
+
+[libraries]
+hivemq-extensionSdk = { module = "com.hivemq:hivemq-extension-sdk" }
```

Comment:
wait, this is now problematic, because when we publish the sdk, there will be no version set for the hivemq-extension-sdk. Before we always set the same version as the version of this sdk itself.

Reply:
added a suggestion of a possible solution

Reply:
Thanks for the catch!